### PR TITLE
update install script and dev environement

### PR DESCRIPTION
**Pull request by MaximeThoonsen**
Hello,

I encountered some issues while installing the project, I think this PR will make the installation smoother. 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [x] Refactoring (no functional changes)
- [ ] Documentation content changes

## Other information

Issue Number: N/A

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,9 +33,7 @@ frontend/start:
 .env: .env.example
 	@if [ -f .env ]; \
 	then\
-		echo '\033[1;41m/!\ The .env.example file has changed. Please check your .env file (this message will not be displayed again).\033[0m';\
-		touch .env;\
-		exit 1;\
+		echo '\033[1;41m/!\ You already have created .env file. You should have a look at the .env.example file to ensure you have set all required environment variables for docker-compose.\033[0m';\
```
Comment (MaximeThoonsen) [inline Makefile]: We shouldn't exit the script if there is already a file. The wording was also not defining what was the problem.

File: docker-compose.yml [hunk 2]
```diff
@@ -4,9 +4,14 @@ services:
   db:
     image: postgres:10
     ports:
-      - "5432:5432"
+      # We changed the default port so people having already a postgres instance running on port 5432 won't have a port conflict
+      - "5433:5432"
```
Comment (MaximeThoonsen) [inline docker-compose.yml]: Simple trick to make the life of people who has already a postgres instance running (like me) easier during the install.

File: docker-compose.yml [hunk 3]
```diff
@@ -4,9 +4,14 @@ services:
...
     volumes:
       - ./postgres-data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=postgres
```
Comment (MaximeThoonsen) [inline docker-compose.yml]: POSTGRES_PASSWORD is mandatory from the doc : https://hub.docker.com/_/postgres
POSTGRES_USER and POSTGRES_DB make the postgres config explicit so it easier for people to understand

File: hooks/pre-commit [hunk 4]
```diff
@@ -34,7 +34,7 @@ else
 fi
 
 # Run black and get the output and return code
-black=$(black --exclude venv --diff --check backend/.)
+black=$(docker-compose exec -T backend black --diff --check .)
```
Comment (MaximeThoonsen) [inline hooks/pre-commit]: If you are using the docker dev env, you may not have black and flake installed

**Comment by phacks** [comment]
@all-contributors please add @MaximeThoonsen for docs!

**Pull request merged**